### PR TITLE
ESP8266OccupancySensor Example

### PR DESCRIPTION
**Pull request by Letsch22**

### CODE REVIEW

**Comment by azdle** [comment]
This looks pretty good!

Two things,
1. Could you add a readme with instructions on how to get it working including notes on what someone would need for tools beyond the base Arduino stuff.
2. Can you make sure your comments are consistant. Things like if there is a space after the //, a period at the end, and whether it starts with and upper case letter. (This is minor, but I'd like to keep everything as clean as possible.)

Thanks

**Comment by Letsch22** [comment]
Cool! I'll clean up with some of your suggestions. I'll add a final commit with a link to the forum post I will make in the readme. Look for that later.

**Comment by azdle** [comment]
Sounds Good

File: examples/ESP8266OccupancySensor/readme.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+	2. Select "I want to create a generic device."
+
+	3. Leave Device Type as generic, enter your timezone, and optionally add a location. You may also limit the resources your device may use.
+
+	4. Name your device.
+
+8. Click on the device you just created to open the Device Information window.
+
+9. Add the two datasources "uptime" and "command". (For Each)
```
Comment (azdle) [inline examples/ESP8266OccupancySensor/readme.md]: We just need one `occupancyint` dataport here, right?

Comment (Letsch22) [inline examples/ESP8266OccupancySensor/readme.md]: Just fixed: need two (the second ''' occupancy ''' for the Lua script)

**Comment by azdle** [comment]
Great, looks good to go.

**Pull request merged**